src/mon/Monitor.cc

```diff
@@ -464,6 +464,7 @@ const char** Monitor::get_tracked_conf_keys() const
     "clog_to_graylog",
     "clog_to_graylog_host",
     "clog_to_graylog_port",
+    "host",
```

Comment:
@tchaikov  Sorry, sir. I misunderstand your wonderful explanation and show no respect for your kind help. I'll fold these into the commit message.